app/models/json_schema.rb

```diff
@@ -3,7 +3,7 @@ class JsonSchema
   class << self
 
     def generate(json_template)
-      schema = JSON::SchemaGenerator.generate 'Claim for crown court defence - claim import', json_template
+      schema = File.read(File.join(Rails.root, 'config', 'claim_schema.json'))
```

Comment:
the arg json_template is never used. as we discussed if we go down the hard coded json schema route this should probably be removed.